sufia.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'select2-rails', '~> 3.5.9'
   spec.add_dependency 'json-schema'
   spec.add_dependency 'oauth'
+  spec.add_dependency 'devise', '~> 4.2'
```

Comment:
The application also hasn't released a new major version in over a year.  Consumers cannot reasonably complain "you bumped a dependency in a major version after a year!"  And if you cannot bump deps with a major version, you literally can **never** do it.  There are no doubt plenty of other dependencies that have jumped major versions in that time with breaking changes that you don't seem to care about.  You might as well complain that `hydra-batch-edit` has bumped a version or that CurationConcerns was introduced *at all*.  Run `git diff v6.0.0 sufia.gemspec` to see everything that's been allowed to change since the last new major version.  They still have to navigate that and this is supposed to signify a **bigger** change, but then requiring an update to Devise is somehow unthinkable?  Not feeling it.